ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphContainer.tsx

```diff
@@ -312,15 +313,22 @@ function NetworkGraphContainer({
     // this is the current filtered model that has not been modified yet
     let filteredNodes: CustomNodeModel[] = [...baseModel.nodes];
     let filteredEdges: CustomEdgeModel[] = [...baseModel.edges];
+
+    // filtering nodes/edges based on selection, edges will be [] by default
+    filteredEdges = selectedNode ? getFilteredEdges(filteredEdges, selectedNode) : [];
+
+    // we want to filter out the DNS flows when viewing "inactive" flows (without DNS flows)
+    if (edgeState === 'inactive') {
+        filteredEdges = removeDNSFlows(filteredEdges);
+    }
+
     // if edgeState is extraneous && there is a selectedNode, add in/egress flows nodes/edges
-    if (edgeState === 'inactive' && selectedNode?.data.type === 'DEPLOYMENT') {
+    if (edgeState !== 'active' && selectedNode?.data.type === 'DEPLOYMENT') {
```

Comment:
Isn't this change, to `!== active`, also in your other open PR ?

Reply:
Not too sure if it was